src/long.js

```diff
@@ -1,4 +1,6 @@
 module.exports = Long;
+module.exports.default = Long;
```

Comment:
If this is not done then the transpiled TS code fails. TS transpiled JS code looks like this:

```js
const long_1 = require("long");
long_1.default.fromNumber(1000000);
```
Thus it expects that the imported module from require have a default property which is of type `Long`. Doing this change makes it work.